src/overlay/SurveyManager.cpp

```diff
@@ -568,6 +586,8 @@ SurveyManager::commandTypeName(SurveyMessageCommandType type)
     {
     case SURVEY_TOPOLOGY:
         return "SURVEY_TOPOLOGY";
+    case SURVEY_TOPOLOGY_EXTENDED:
```

Comment:
We should probably cleanup the XDR: there are more than 1 enum.

One contains the list of "commands", and the only supported command is `SURVEY_TOPOLOGY`.

Then there are different `SurveyResponseBody` versions (this PR adds 1 new one), so we should just create a new enum for that `SurveyResponseType` with values `SURVEY_TOPOLOGY_RESPONSE_V0=0` and `SURVEY_TOPOLOGY_RESPONSE_V1=1`

As for this function we can just call `xdr::xdr_traits<SurveyMessageCommandType>::enum_name(type)` instead in `getMsgSummary`

Reply:
Ok, it's updated. I will submit a PR in the vNext repo when this PR is approved.